src/app/client/src/app/modules/core/components/prominent-filter/prominent-filter.component.spec.ts

```diff
@@ -68,6 +68,19 @@ describe('ProminentFilterComponent', () => {
     component.applyFilters();
     expect(router.navigate).toHaveBeenCalledWith([undefined], { queryParams: queryParams });
   });
+
+  it('Filter results should not change on call of  apply filters for same params', () => {
+    const router = TestBed.get(Router);
+    component.formInputData = {
```

Comment:
can't we say component.formInputData = queryParams = { 'gradeLevel': ['Grade 1', 'Grade 2'], 'medium': ['English'] }

Reply:
it is throwing error